src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/sort/SortClause.java

```diff
@@ -28,6 +28,24 @@
               """)
 public record SortClause(@Valid List<SortExpression> sortExpressions) implements SchemaValidatable {
 
+  public boolean isEmpty() {
+    return sortExpressions == null || sortExpressions.isEmpty();
+  }
+
+  public List<SortExpression> tableVectorSorts() {
```

Comment:
oh, tableVectorSorts means vector sort on a vector table column. and table column can't be $vector and $vectorize?